spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaDeploymentProperties.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.eureka.server;
+
+import java.util.Objects;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for the Eureka deployment.
+ *
+ * @author Weix Sun
+ */
+@ConfigurationProperties("eureka")
```

Comment:
Please change the suffix to `eureka.deployment`.

Reply:
Everyone has been using `eureka.environment` and `eureka.datacenter` for a long time, and now that we've changed the prefixes, should we remind everyone in the doc or in the release notes?

Reply:
> Please change the suffix to `eureka.deployment`.

Perhaps we can do the configuration item expiration specification in `spring-configuration-metadata.json` like Spring Boot and recommend the configuration item with the `eureka. deployment` prefix instead.

Is it worth it？😂

Reply:
You're right - sorry for the confusion, but given what  you've pointed out, I think we should switch it back to `eureka` prefix and then rename the class to `EurekaProperties` . We might want to do some additional work on all SC Netflix properties prefixes later on to ensure more consistency, but that will have to be done on the `main` branch.

Reply:
> I think we should switch it back to `eureka`prefix and then rename the class to `EurekaProperties`

Ok, Let me do a quick fix